docs/markdown/intro/README.md

```diff
@@ -79,7 +79,11 @@ BootstrapVue pre-compiled CSS files by setting the folliwing option(s) to `false
 
 ```js
 {
-  modules: [['bootstrap-vue/nuxt', { bootstrapCss: false, bootstrapVueCss: false }]]
+  modules: ['bootstrap-vue/nuxt'],
+  bootstrapVue: {
+    bootstrapCSS: false, // or `css`
+    bootstrapVueCSS: false // or `bvCSS`
```

Comment:
I just setup this plugin and wasn't getting any CSS until I explicitly set these to `true`. Is that the expectation? 

Based on the changes here it seems like the previous default was that these options defaulted to `true` but now, AFAICT, there is no default?

Reply:
This will be fixed in the next release via #2701